packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -81,12 +85,7 @@
   height: var(--icon-size);
 }
 .list-item .icon > svg *:not([fill='none']) {
-  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
-  fill: var(--color-for-navbar-icon);
-}
-.list-item .icon__active > svg *:not([fill='none']) {
-  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
-  fill: var(--color-for-navbar-icon-when-active);
+  fill: var(--color-surface);
```

Comment:
Changed the icon color. The icon also doesn't change color anymore if active state.